apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -116,6 +143,6 @@ private static string GetEnvironmentVariableOrDefault(string name, string defaul
             return string.IsNullOrEmpty(value) ? defaultValue : value;
         }
         
-        public SpannerConnection GetConnection() => new SpannerConnection(ConnectionString);        
+        public SpannerConnection GetConnection() => new SpannerConnection(Builder);
```

Comment:
Why is this change required? I'd expect the connection string to be fine.

Reply:
Or is this because we've set the SessionPoolManager because of the options? It feels like a lot of this change could go away if we didn't need to set the options explicitly.